packages/vega-typings/types/spec/transform.d.ts

```diff
@@ -86,6 +86,7 @@ export interface AggregateTransform {
   groupby?: FieldRef[] | SignalRef;
   fields?: (FieldRef | null)[] | SignalRef;
   ops?: (AggregateOp | SignalRef)[] | SignalRef;
+  aggregate_params?: object[];
```

Comment:
Shouldn't this be `number[]`?

Reply:
It should take any sort of input. Different ops will take different parameters. "exponential" takes a number, but "cook_breakfast" might take a list of string ingredients, and "calculate_gravitational_field" might take a complicated object describing the local spacetime field.

Reply:
Let's change it when we actually have aggregates that use those types.